libs/api/src/index.ts

```diff
@@ -34,32 +34,3 @@ export * from './lib/challenges/algebraicCaptchaChallenge'
 export * from './lib/challenges/cfTurnstile'
 
 export {getUserForCredentials} from './lib/graphql/user/user.queries'
-
-export * from '@wepublish/banner/api'
```

Comment:
Removed this as we don't really do packages anymore. We also have Membership V2 model that would clash